src/components/templates/PartyMap/components/ScheduleItem/ScheduleItem.tsx

```diff
@@ -29,7 +29,7 @@ export const ScheduleItem: React.FunctionComponent<PropsType> = ({
   const dispatch = useDispatch();
 
   const schedulePrimaryClasses = classNames({
-    ScheduleItem--primary: isCurrentEvent,
+    "ScheduleItem--primary": isCurrentEvent,
```

Comment:
Is this change correct? It was throwing an error without the quotes

Reply:
oops, yeah, didn't account that. Sorry. 🙈